### PR TITLE
Make migration launch command working with all publishable formats

### PR DESCRIPTION
**Pull request by lqueiroga**
* Create image with targetFormat and installProfile
* Attach the image to GenerationStage
* Attach os family to ScanStage
* Verify if os value is linux
* Verify if aws or outscale builder contains installation with diskSize

### CODE REVIEW

File: hammr/commands/migration/migration.py [hunk 1]
```diff
@@ -83,12 +83,16 @@ def do_launch(self, args):
...
             migration_config = migration_utils.retrieve_migration_configuration(do_args.file)
             target_format = migration_utils.retrieve_target_format(self.api, self.login, migration_config["target"]["builder"]["type"])
+            image = migration_utils.retrieve_image(migration_config["target"]["builder"], target_format, self.api, self.login)
+            if image is None:
+                return 2
             publish_image = migration_utils.retrieve_publish_image(migration_config["target"]["builder"], target_format)
             if publish_image is None:
                 return 2
             cred_account = migration_utils.retrieve_account(self.api, self.login, migration_config["target"]["builder"]["account"]["name"])
+            publish_image.credAccount = cred_account
```
Comment (segalaj) [inline hammr/commands/migration/migration.py]: why did you move this line?

File: hammr/utils/migration_utils.py [hunk 2]
```diff
@@ -54,6 +55,8 @@ def check_mandatory_migrate(migration):
         raise Exception("check yours parameters in file, no attribute [name] for [migration]")
     if not "os" in migration:
         raise Exception("check yours parameters in file, no attribute [os] for [migration]")
+    elif migration["os"] != "linux":
+        raise Exception("check yours parameters in file, attribute [os] for [migration] is not correct. Only linux is valid")
```
Comment (segalaj) [inline hammr/utils/migration_utils.py]: Maybe: ```Only 'linux' is supported```

File: hammr/commands/migration/migration.py (hunk 1 above)
Comment (lqueiroga) [reply hammr/commands/migration/migration.py]: In fact for me it was not useful to give credAccount as parameter of create_migration method just to set it in publishImage.

File: hammr/utils/migration_utils.py (hunk 2 above)
Comment (lqueiroga) [reply hammr/utils/migration_utils.py]: agree

File: tests/unit/utils/test_migration_utils.py [hunk 3]
```diff
@@ -306,6 +317,128 @@ def test_retrieve_target_format_raise_exception_when_the_target_format_not_found
...
+
+    def test_set_install_profile_disk_size_raise_exception_when_no_diskSize(self):
+        # given
+        install_profile = uforge.InstallProfile()
+        install_profile.diskSize = 0
+
+        builder = {
+            "type": "Amazon AWS",
+            "installation": {
+                "noDiskSize": 12
```
Comment (cinlloc) [inline tests/unit/utils/test_migration_utils.py]: Not sure to understand this test
